documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -770,17 +770,18 @@ $ sudo raspi-config nonint do_boot_rom <E1/E2>
 
 ==== Wayland
 
-Switch between the X11 and Wayland backends. Raspberry Pi 4 and later use Wayland by default; other models of Raspberry Pi use X11 by default.
+Switch between the X11 and Wayland backends, and choose a window manager. Since Raspberry Pi OS _Bookworm_, all Raspberry Pi models run Wayland using the labwc window manager by default.
+
+NOTE: To use Wayland on Raspberry Pi models prior to Raspberry Pi 4 running a version of Raspberry Pi OS earlier than _Bookworm_, add `wayland=on` to `/boot/firmware/cmdline.txt`.
```

Comment:
See above. Also, I never actually edited `/boot/firmware/cmdline.txt` myself (when I was testing labwc on older Pis) - I just used `raspi-config` and let that make any necessary changes for me :slightly_smiling_face: